NEW_README.md

```diff
@@ -6,8 +6,15 @@ At the end of the run, the gatling results and atop results will be copied back
 
 * Clone the gatling-puppet-load-test repo locally: https://github.com/puppetlabs/gatling-puppet-load-test
 * cd into the gatling-puppet-load-test root directory
-* For apples to apples runs, you can use the checked-in hosts files: pe-perf-test.cfg or foss-perf-test.cfg (default for the performance rake task).
-* In order for us to take advantage of the new AWS account with access to internal network resources, you need to use ABS. The 'performance' task will automatically use ABS to provision 2 AWS instances and then execute tests against those instances.
+* For apples to apples runs, you can use the checked-in hosts files: [pe-perf-test.cfg](config/pe-perf-test.cfg) or [foss-perf-test.cfg](/config/foss-perf-test.cfg) (default for the performance rake task).
```

Comment:
The part about (default for the performance rake task) doesn't work out.  There isn't one default (it wasn't clear which it was saying was the default anyway).  The default is picked based on BEAKER_INSTALL_TYPE.  You didn't technically add the default part... :)  But you happened to be editing the same line. :)

Reply:
All I changed here was to add markdown to make pe-perf-test.cfg or foss-perf-test.cfg links to the files.

Reply:
Even better. :)